staging/operator-registry/go.mod

```diff
@@ -66,13 +65,5 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace (
-	// Currently on a fork for two issues:
-	// 1. stage registry proxy didn't like requests with no scopes, see https://github.com/containerd/containerd/pull/4223
-	// 2. prod registry proxy returns a 403 on post, see https://github.com/containerd/containerd/pull/3913
-	// The fork can be removed when both issues are resolved in a release, which should be 1.4.0
-	github.com/containerd/containerd => github.com/ecordell/containerd v1.3.1-0.20200629153125-0ff1a1be2fa5
-
-	// latest tag resolves to a very old version. this is only used for spinning up local test registries
-	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
-)
+// latest tag resolves to a very old version. this is only used for spinning up local test registries
+replace github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```

Comment:
@njhale should we drop this for this backport and revert this to the original?

Reply:
@perdasilva the cherry-pick is fine -- we should update the root go.mod and call `make vendor` from the root directory.